tests/ctl/core/test_config.py

```diff
@@ -17,6 +17,13 @@
 }
 
 
+@pytest.mark.unit
+def test_get_config_verbose() -> None:
+    """Simple test to check the 'verbose' code path."""
+    config = get_config(verbose=True, config_path_override="fakefiledoesntexist.toml")
+    assert config
```

Comment:
Nit: We're not actually checking that the `verbose` kwarg does anything

Reply:
That feels ok to me on such a simple implementation here, we'd notice pretty quickly